entries.go

```diff
@@ -157,6 +157,52 @@ func GetAllEntries() ([]Item, error) {
 	return allEntries, err
 }
 
+// GetAllEntriesFiltered calls GetEntries with the given archive and starred parameters parameters, thus returning all articles as []wallabago.Item which match the filter
+func GetAllEntriesFiltered(archive, starred *bool) ([]Item, error) {
```

Comment:
i have some doubt with duplicating all the code for the different uses GetEntries calls. Maybe it is better to extend the parameter list of this GetAllEntriesFiltered func to all from GetEntries except the page, which would mean `bodyByteGetterFunc BodyByteGetter, archive int, starred int, sort string, order string, perPage int, tags string`. 
Afterwards the GetAllEntries func could be refactored to use this GetAllEntriesFiltered. And finally we wouldn't have so much duplicated code.